README.md

```diff
@@ -93,7 +93,25 @@ Each service has its own specific role and communicates via REST APIs.
 
 ![Spring Petclinic Microservices architecture](docs/microservices-architecture-diagram.jpg)
 
+## Integrating the Spring AI Chatbot
```

Comment:
**suggestion**: we need to offer a solution to developers who don't have an Azure or OpenAI account and don't want to pay for it: either enable fallback by default, or disable chat, or have a local solution with ollama. What do you think @odedia ?

Reply:
I began with using Llama 3.1, but the chat client became incredibly verbose and not as useful as OpenAI, that's why I switched to it.
![image](https://github.com/user-attachments/assets/ef72677d-a2a5-4d10-b2a0-b3eb03d78ee0)

Reply:
@odedia I've switched the Maven configuration from Azure OpenAI to OpenAI with the free `demo` account that OpenAI suggests. To use it, we have to use the `gpt-4o-mini` model.